hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/java/org/apache/hadoop/fs/http/server/HttpFSServerWebServer.java

```diff
@@ -119,6 +120,7 @@
       conf.set(HttpServer2.FILTER_INITIALIZER_PROPERTY, actualInitializers);
     }
 
+    DefaultMetricsSystem.initialize("httpfs");
```

Comment:
We probably want to do the same for kms..

Reply:
Thank you. Added https://github.com/apache/hadoop/pull/2145/commits/2f3db465ccd6b1741eb1c2a363bdffa7639032fe